.eslintrc.json

```diff
@@ -1,3 +1,8 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"]
+  "extends": ["next/core-web-vitals", "prettier"],
+  "plugins": ["@typescript-eslint"],
+  "parser": "@typescript-eslint/parser",
+  "rules": {
+    "@typescript-eslint/no-unused-vars": "error"
+  }
```

Comment:
We add the typescript specific parser because we're using typescript. This stops the linter complaining that our types have unused variables.

Reply:
very nice